spec/fixtures/eurofxref-daily.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Where is this file from? Is it manually downloaded? How can be updated in future?

Have you consider to use [vcr](https://github.com/vcr/vcr) for automated downloading and updating source files?